install/ansible/README.md

```diff
@@ -0,0 +1,110 @@
+Table of Contents
+=================
+
+   * [Ansible playbook to install Istio](#ansible-playbook-to-install-istio)
+      * [Prerequisites](#prerequisites)
+      * [Execution](#execution)
+      * [Typical use cases](#typical-use-cases)
+
```

Comment:
can you actually move this documentation to istio.github.io under the kube setup? Thats where people go for installing istio

Reply:
No problem.
Do you want me to move the whole readme or just have a section that points this README?